Lifecycle.md

```diff
@@ -34,12 +34,12 @@ chartered by the TC of the project they exist under.
 
 In general, a TLP or TLWG (hereafter TL*) will have the following lifecycle:
 
-	1. Like-minded individuals form a TC in order to establish a TL*
-	2. That TC establishes a charter for their TL*
-	3. That TC submits an application to the Planetary Software TSC
-	4. If the application is accepted by the TSC, the TL* enters Incubation
-	5. The TSC works with the TC to get the TL* graduated from Incubation
-	6. The TL* is now a fully authorized Planetary Software TL*
+	* Like-minded individuals form a TC in order to establish a TL*
+	* That TC establishes a charter for their TL* and submits an application to the Planetary Software TSC
+	* If the application is accepted by the TSC, the TL* enters Incubation
+	* A mentor(s) is assigned tor work the TC to get the TL* to ensure requirements are met
+	* Once requirements are met, the TSC votes to graduate the TL* from Incubation
+	* The TL* is now a fully authorized Planetary Software TL*
```

Comment:
I know these were like this in the parent document (but I think that was an oversight), these probably don't need to be verbatim text, and probably can be 'regular'.

Reply:
OK. Do we care about numbering these steps or just having them in order? I originally removed the numbers to not confuse with the numbered steps I added below.

Reply:
I don't think so.  If you think they look better as bullets, go for that.

Reply:
Ok. Changed but kept bullets.